code/modules/cargo/packs.dm

```diff
@@ -285,6 +285,18 @@
 					/obj/item/ammo_box/c38/iceblox)
 	crate_name = "ammo crate"
 
+/datum/supply_pack/security/nemesis
+	name = "Nemesis Solutions Equipment"
+	desc = "Contains full equipment for one security officer and a high-tech combat vest from Nemesis Solutions along with some gadgets for them to use."
+	cost = CARGO_CRATE_VALUE * 4
+	access_view = ACCESS_ARMORY
+	contains = list(/obj/item/clothing/suit/armor/vest/nemesis,
+					/obj/item/storage/box/nemesis_equipment,
```

Comment:
since the items from this box are consumable, you should probably allow it to be purchased separately from the rest of the kit